run-coverage-test.sh

```
@@ -0,0 +1,4 @@
+#!/bin/bash
+cd /Users/a.fouad/S/New_elzatona
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)